apex/contrib/csrc/group_norm/group_norm_nhwc.cpp

```diff
@@ -867,6 +867,20 @@ int main(int argc, char **argv) {
   Group_norm_nhwc_bwd_params params_bwd;
   memset(&params_bwd, 0, sizeof(params_bwd));
 
+  PrecisionMode precision;
+  if (use_fp32)
+  {
+    precision = PrecisionMode::FP32IOFP32W;
+  }
+  else if (use_bf16)
+  {
+    precision = PrecisionMode::BF16IOFP32W;
+  }
+  else
+  {
+    precision = PrecisionMode::FP16IOFP32W;
+  }
```

Comment:
nit-picking as `precision` doesn't seem to be modified after this initialization w/ if-else
```suggestion
  const auto precision = [&]() -> PrecisionMode {
    if (use_fp32) {
      return PrecisionMode::FP32IOFP32W;
    } else if (use_bf16) {
      return PrecisionMode::BF16IOFP32W;
    } else {
      return PrecisionMode::FP16IOFP32W;
    }
  }();
```